tokens/global/space.json5

```diff
@@ -0,0 +1,202 @@
+{
+  space: {
+    'eighth-x': {
+      value: '2px',
+      category: 'space',
+    },
+    'quarter-x': {
+      value: '4px',
+      category: 'space',
+    },
+    'half-x': {
+      value: '8px',
+      category: 'space',
+    },
+    'three-quarter-x': {
+      value: '12px',
+      category: 'space',
+    },
+    'one-x': {
+      value: '16px',
+      category: 'space',
+    },
+    'one-and-a-half-x': {
+      value: '24px',
+      category: 'space',
+    },
+    'two-x': {
+      value: '32px',
+      category: 'space',
+    },
+    'four-x': {
+      value: '64px',
+      category: 'space',
+    },
+    inline: {
+      'eighth-x': {
+        value: '{space.eighth-x.value}',
+        category: 'space',
+      },
+      'quarter-x': {
+        value: '{space.quarter-x.value}',
+        category: 'space',
+      },
+      'half-x': {
+        value: '{space.half-x.value}',
+        category: 'space',
+      },
+      'three-quarter-x': {
+        value: '{space.three-quarter-x.value}',
+        category: 'space',
+      },
+      'one-x': {
+        value: '{space.one-x.value}',
+        category: 'space',
+      },
+      'one-and-a-half-x': {
+        value: '{space.one-and-a-half-x.value}',
+        category: 'space',
+      },
+      'two-x': {
+        value: '{space.two-x.value}',
+        category: 'space',
+      },
+      'four-x': {
+        value: '{space.four-x.value}',
+        category: 'space',
+      },
+    },
+    inset: {
+      'eighth-x': {
+        value: '{space.eighth-x.value}',
+        category: 'space',
+      },
+      'eighth-x-squish': {
+        value: '0 2px',
+        category: 'space',
+      },
+      'eighth-x-stretch': {
+        value: '4px 2px',
```

Comment:
Tokens are meant to be single values. It doesn't know how to handle multiple values like this.

We may be able to add something for it...but I don't know if android/ios can accept multiple values this way.

Reply:
iOS and Android definitely can't handle that `multiple value` format.

Reply:
Maybe in the transform, this can map to vertical(top/bottom) and horizontal(left/right) variants.  I guess alternatively could define them separately and then combine them in the transform to make the web format.  I prefer the later.

Reply:
@KrisKnabel what would that look like? potentially we could have two values for each top/bottom left/right and then combine them in the web dir to be the intended value?

Reply:
The way that design thinks about stretch,squish,inset,inline,stack, is confusing and does not map to how iOS and Android lay out components.  I talked with Corey and he still has not figured out how he will be representing component layout in his designs.  He is meeting with Duc the Android developer next week to try to come up with something.  I think that some of spacing could be useful to iOS/Android, but only the 1/8x - 8x values.  The stretch/squish/inset/inline/stack/ just does not make sense for mobile.  So maybe those should just go in the Web directory.